### PR TITLE
@orta => use serif font for prices on non-auction works

### PR DESCRIPTION
**Pull request by 1aurabrown**
closes https://github.com/artsy/eigen/issues/25. I also updated the podfile to reflect my new computer's username. Would be cool to have a better way of deciding which pods to install.

### CODE REVIEW

**Comment by orta** [comment]
Failures are legit

**Comment by 1aurabrown** [comment]
@katarinabatina just checking that these changes look correct to you (see images)

File: Artsy/Classes/Views/ARArtworkPriceView.m [hunk 1]
```diff
@@ -28,17 +28,21 @@ - (void)updateWithArtwork:(Artwork *)artwork andSaleArtwork:(SaleArtwork *)saleA
         row.messageLabel.textColor = [UIColor artsyRed];
         row.messageLabel.font = [UIFont sansSerifFontWithSize:row.messageLabel.font.pointSize];
         row.messageLabel.text = @"SOLD";
-    } else if (saleArtwork != nil) {
-        row.messageLabel.text = @"Buy Now Price:";
+        if (saleArtwork != nil) {
+            row.priceLabel.font = [UIFont sansSerifFontWithSize:24];
```
Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkPriceView.m]: for AUCTION works use sans serif

File: Artsy/Classes/Views/ARArtworkActionsView.m [hunk 2]
```diff
@@ -76,7 +76,7 @@ - (void)updateUI
         shownPrice = YES;
     } else if ([self showPriceLabel]) {
         self.priceView = [[ARArtworkPriceView alloc] initWithFrame:CGRectZero];
-        [self.priceView updateWithArtwork:self.artwork];
+        [self.priceView updateWithArtwork:self.artwork andSaleArtwork:self.saleArtwork];
```
Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: need to pass the sale artwork in order for the priceView to correctly identify artworks which have been sold && at autiong

**Comment by orta** [comment]
@katarinabatina if you're happy, hit this button twice: 
screen shot 2015-01-30 at 12 44 48 pm

**Comment by 1aurabrown** [comment]
@orta :green_apple:

**Pull request merged**